bpf/lib/nat46.h

```diff
@@ -401,4 +410,38 @@ static inline int ipv6_to_ipv4(struct __sk_buff *skb, int nh_off,
 	return 0;
 }
 
+__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT64) int tail_ipv6_to_ipv4(struct __sk_buff *skb)
```

Comment:
I think you should move this function into `bpf_lxc.c` because `CILIUM_CALL_IPV4` which is the tail call you make next is only available in that context. We want to avoid somebody making a tail call to `CILIUM_CALL_NAT64` from outside lxc context.